tailwind.config.js

```diff
@@ -8,6 +8,8 @@ var dir = path.resolve(__dirname, 'src/plugins')
 module.exports = {
   // Our own prefix.
   prefix: 'su-',
+  // Enable dark mode utility classes
+  darkMode: 'media',
```

Comment:
I enabled dark mode classes since it's getting more popular. It does increase the file size by a lot, but removing xs = 0 breakpoint does help a bit (about 1/6 I guess)

Reply:
> The media strategy uses the `prefers-color-scheme` media feature under the hood.

We should think on that a bit and decide if we want to support those systems and browsers with this feature and rely on the users' preference or to have it as a toggle with class. I am leaning towards `media` but we should put more thought into it.

Reply:
Yeah I'm thinking about that too. Media feels more true to the TW spirit, but of course the file size is big. Browser support is really quite good.

Reply:
Let's leave it as is. We might have to explore the tailwind themes at some point and from what I can tell they tend to use the class name switcher. Something to bring up with Kerri. Either approach still requires the developer to place the css classes on each of the components.

Reply:
Ok I'll get rid of it. We can talk about the TW reduce motion stuff later too 😄